tests/test_service.py

```diff
@@ -231,3 +233,9 @@ def test_tx_id_optional(self):
         del message['tx_id']
 
         self.assertValid(message)
+
+    def test_completed_not_boolean_fails(self):
+        message = json.loads(self.message['0.0.1'])
+        message['completed'] = ''
```

Comment:
Is it possible to check whether it works with the key missing from the dict altogether? (Just in case a 'null' boolean can be treated as valid, but `false` for example)